### PR TITLE
chore: add fantasy-08/CodersArena

### PR DESCRIPTION
**Pull request by fantasy-08**

### CODE REVIEW

File: data/projects.json [hunk 1]
```diff
@@ -68,5 +68,15 @@
...
         "tags": "tic-tac-toe, c-projects, game, c++"
+    },
+     {
+        "name": "CodersArena",
+        "display": "Coders Arena",
+        "description": "A (web application) platform to have a coing combat with your friend (30 minutes) with random question from our library and also a platform to create test (MCQ and time based) and also to attend MCQ test.",
+        "author": "fantasy-08",
+        "category": "website",
+        "level": "intermediate",
+        "site": "https://github.com/fantasy-08/CodersArena",
```
Comment (ming-tsai) [inline data/projects.json:79]: Hi @fantasy-08,
This is not a website, left it on an empty string.
```suggestion
        "site": "",
```

Comment (fantasy-08) [reply data/projects.json:79]: http://boringcoder.herokuapp.com/
This is the website

**Review by fantasy-08 — COMMENTED** [review]
http://boringcoder.herokuapp.com/
this is the deployed site

**Review by ming-tsai — APPROVED** [review]
Hi @fantasy-08,
Nice work, and welcome to be part of the team
> The invitation [link](https://github.com/MakeContributions)

**Pull request merged**